http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -358,7 +358,8 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       config.defaultBlockSizeBytes,
       None,
       None,
-      labels
+      labels,
+      None
```

Comment:
Is there something else you want me to do here?

Reply:
this is fine since we're not supporting creating a runtime from a source disk directly